README.md

```diff
@@ -1,12 +1,11 @@
 # bobthefish
 
-bobthefish is a Powerline-style, Git-aware fish theme optimized for awesome.
+`bobthefish` is a Powerline-style, Git-aware [Oh My Fish][omf] theme optimized for awesome.
```

Comment:
technically, it's not. it's a fish theme. you can install it without omf, and it has no dependencies on omf. that just happens to be the easiest way to get it installed :)